localstack/utils/common.py

```diff
@@ -992,3 +992,19 @@ def parallelize(func, list, size=None):
     pool.close()
     pool.join()
     return result
+
+
+def generate_random_revision_id():
```

Comment:
We can use the built-in [`uuid`](https://docs.python.org/3/library/uuid.html) module to generate unique UUIDs:
```
str(uuid.uuid4())
```

Can we move `generate_random_revision_id(..)` into `lambda_api.py` (as it seems to be Lambda specific), and use the `uuid` module instead of manually generating the ID.